src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,139 @@
+              <Loader />
+            )
+          }
+
+          {
+            isError && (
+              <p data-cy="peopleLoadingError" className="has-text-danger">
+                Something went wrong
+              </p>
+            )
+          }
+
+          {
+            !isError && !isLoading && !people.length && (
+              <p data-cy="noPeopleMessage">
+                There are no people on the server
+              </p>
+            )
+          }
+
+          {
+            !isError && !isLoading && !!people.length && (
+              <table
+                data-cy="peopleTable"
+                className="table is-striped is-hoverable is-narrow is-fullwidth"
+              >
+                <thead>
+                  <tr>
+                    <th>Name</th>
+                    <th>Sex</th>
+                    <th>Born</th>
+                    <th>Died</th>
+                    <th>Mother</th>
+                    <th>Father</th>
+                  </tr>
+                </thead>
+
+                <tbody>
+                  {
+                    people.map(person => {
+                      const {
+                        sex,
+                        born,
+                        died,
+                        fatherName,
+                        motherName,
+                        slug,
+                      } = person;
+
+                      const mother = people
+                        .find(mom => mom.name === motherName);
+                      const father = people
+                        .find(dad => dad.name === fatherName);
+
+                      return (
+                        <tr
+                          data-cy="person"
+                          className={classNames({
+                            'has-background-warning': slug === selectedSlug,
+                          })}
+                        >
+                          <td>
+                            <PersonLink person={person} />
+                          </td>
+
+                          <td>{sex}</td>
+                          <td>{born}</td>
+                          <td>{died}</td>
+                          <td>
+                            {
+                              (mother && (
+                                <PersonLink person={mother} />
+                              ))
+                              || (motherName && (
+                                motherName
+                              ))
+                              || '-'
+                            }
+                          </td>
+                          <td>
+                            {
+                              (father && (
+                                <PersonLink person={father} />
+                              ))
+                              || (fatherName && (
+                                fatherName
+                              ))
+                              || '-'
+                            }
```

Comment:
```suggestion
                            {
                              father ? (
                                <PersonLink person={father} />
                              )
                              : fatherName || '-'
                            }
```
I think it's looks better